draft_release_notes.md

```diff
@@ -0,0 +1,14 @@
+```markdown
+## <Feature> added to DT-UI
+**Description of Previous Approach**
+
+**Description of New Approach**
+
+**Benefits of New Approach**
+
+**Integration**
+[USAS/Broker PR here](https://github.com/fedspendingtransparency/usaspending-website/pull/1431):
+
+- [] Development Complete
```

Comment:
Wouldn't development already have been complete upon PR creation?

Reply:
What is the reasoning behind having tasks in the release notes, shouldn't these be moved or are already included in the PR checklist.

Reply:
B/c we might reasonably expect the release notes to be created before the development is done.

Reply:
I do not believe release notes should have a checklist. Release notes should be informing users of changes.  We could add a check list for release notes PRs.

Reply:
Understood!!